docs/examples/datasets/pipelines.py

```diff
@@ -350,3 +350,91 @@ def test_multiple_pipelines(client):
     )
     assert df.shape == (4, 4)
     assert found.sum() == 4
+
+
+# docsnip aggregate_with_str_window
+@meta(owner="data-eng-oncall@fennel.ai")
```

Comment:
We recently added support for specifying `__owner__` at module level so that this doesn't need to be specified for each dataset. That makes the documentation feel slightly less verbose. 

Assuming this docsnip is going in docs, could you please do that here too?

Reply:
Got it. Should I go ahead and remove all meta from existing datasets in the file?

Reply:
You can as long as the documentation itself isn't selectively explaining the concept of ownership (in which case we want to keep it). Also, in some places, the description of code snippets refers to line numbers so if you remove this line, the line numbers may change - if that happens, you'd need to modify the documentation text as well.

Reply:
got it